Automattic/simplenote-android
# Remove Google Analytics

We're no longer using Google Analytics, so say goodbye! 👋 

**To Test**
Build the app, verify things work as expected and that tracks events still fire.
